app/controllers/application_controller.rb

```diff
@@ -6,6 +6,10 @@ class ApplicationController < ActionController::Base
   private
 
   def count_items
-    @item_numbers = Item.count
+    @item_numbers = redis.get('items_number') || 0
+  end
+
+  def redis
+    @redis ||= Redis.new
```

Comment:
check this redis initialization, please 
https://hixonrails.com/ruby-on-rails-tutorials/ruby-on-rails-redis-installation-and-configuration/
and use approach with `Redis.current`